types/fetch-mock/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "strict-export-declare-modifiers": false
```

Comment:
On `master` there are 17 issues of this kind for `fetch-mock`. My change brings this down to three (`declare namespace`, `declare var fetchMock` and `export = fetchMock`). I don’t think you can avoid this three instances, can you? I can remove the rule and disable it for those three lines though.